src/source/reference/get-feed-import-id-detail-vendor-id.rst

```diff
@@ -1,6 +1,10 @@
 .. index:: GET /feed/import/{id}/detail/{vendorId}
 .. _get_feed_import_id_detail_vendor_id:
 
+.. warning::
+
+	This call is scheduled to be deprecated. Please use :ref:`_get_feed_import_id_detail` instead.
```

Comment:
the reference is not resolved correctly for me locally, @twessling-icas what I have done wrong?

Reply:
fixed it